Sources/MediaViewer/PreviewItemView/Builtins/PlayerPreviewItemViewController.swift

```diff
@@ -1,51 +1,66 @@
 import UIKit
 import AVKit
+import VideoToolbox
 import os
 
 fileprivate let logger = Logger(
     subsystem: Bundle.main.bundleIdentifier! + ".logger",
     category: #file
 )
 
-public final class PlayerPreviewItemViewController: UIViewController {
-    private let playerView = AVPlayerView()
-    
-    let player: AVPlayer
+@MainActor
+public final class PlayerPreviewItemViewController: AVPlayerViewController {
+    private var playerOutput: AVPlayerItemVideoOutput
```

Comment:
Why is the a var? Where is it changed? 🤔

Reply:
removed. it's from a previous attempt at getting thumbnail.